ts/node/migration/sessionMigrations.ts

```diff
@@ -1854,6 +1886,58 @@ function updateToSessionSchemaVersion32(currentVersion: number, db: BetterSqlite
   console.log(`updateToSessionSchemaVersion${targetVersion}: success!`);
 }
 
+function updateToSessionSchemaVersion33(currentVersion: number, db: BetterSqlite3.Database) {
+  const targetVersion = 33;
+  if (currentVersion >= targetVersion) {
+    return;
+  }
+
+  console.log(`updateToSessionSchemaVersion${targetVersion}: starting...`);
+  db.transaction(() => {
+    db.exec(`ALTER TABLE ${CONVERSATIONS_TABLE} ADD COLUMN blocksSogsMsgReqsTimestamp INTEGER;`);
+
+    const loggedInUser = getLoggedInUserDuringMigration(db);
+    const ourConversation = getLoggedInUserConvoDuringMigration(db);
```

Comment:
I'm wondering if `getLoggedInUserConvoDuringMigration` should not return an object containing the keys and our conversation so that we only have to do 1 function call in the case we want to check for logged in and get the conversation. Calling both functions means we run some of the same code twice. We can just return null if it fails as you have already